templates/shared/combomarkdowneditor.tmpl

```diff
@@ -3,19 +3,22 @@ Template Attributes:
 * ContainerId: id attribute for the container element
 * ContainerClasses: additional classes for the container element
 * MarkdownPreviewUrl: preview url for the preview tab
+* MarkdownPreviewMode: the preview mode (default to comment)
```

Comment:
1. What other values are possible?
2. **Why** are other values possible?
3. How do these values differ, and why should these differences exist?

Reply:
As far as I know, possible modes are `markdown`, `comment`, `gfm` and `file`. The default is `comment`. I needed to use `markdown` instead, because `comment` renders the issue reference.

Reply:
Should at least document all possible values. It does seem like a especially obscure kind of option.